deployment/base/rbac/scc.yaml

```diff
@@ -0,0 +1,26 @@
+# TODO: check if we can reduce scc capabilities
```

Comment:
can we use one of the default OCP SCCs? If memory serves, we do have a builtin SCC very similar to this

Reply:
Ack, will look into this.

Reply:
We can run it with `privileged` SCC. Updated to that.

Reply:
Here is what privileged SCC looks like:
```
NAME                              PRIV    CAPS         SELINUX     RUNASUSER          FSGROUP     SUPGROUP    PRIORITY     READONLYROOTFS   VOLUMES

privileged                        true    ["*"]        RunAsAny    RunAsAny           RunAsAny    RunAsAny    <no value>   false            ["*"]

```